lib/nice_partials/partial/section.rb

```diff
@@ -20,7 +20,7 @@ def content?
   private
 
   def write_content_for(content = nil, &block)
-    if content && !pending?
+    if content.class <= String && !pending?
```

Comment:
Rails 7.1 is gonna ship with a `ActiveSupport::SafeBuffer` class that doesn't inherit from String so I don't think this will work in the future, potentially.

I don't understand why we need this after the `has_capturing_yield?` fix?

Reply:
> Rails 7.1 is gonna ship with a `ActiveSupport::SafeBuffer` class that doesn't inherit from String

I knew I had read that somewhere! I mentioned it in the commit message, but had a funny feeling that would be outdated soon enough.

You're right, after the `has_capturing_yield?` change, this line is no longer necessary.